src/tools/sdk/dart_sdk_archive/lib/src/util.dart

```diff
@@ -39,6 +39,7 @@ const Map<String, String> archiveMap = {
   'Windows': 'windows',
   'ia32': 'ia32',
   'x64': 'x64',
+  'arm64': 'arm64',
   'ARMv7': 'arm',
   'ARMv8 (ARM64)': 'arm64',
```

Comment:
It's a little confusing to see both arm64 & ARM64 in the table. I supposed you only see it when **OS** is **All**, but... can we make them more consistent?

![image](https://user-images.githubusercontent.com/2164483/126386518-d172cba3-1d68-4601-bfd1-00c66b4c4b4f.png)

Reply:
We can certainly change `arm64` to `ARM64` on macOS, and it looks like Apple also uses that:
https://developer.apple.com/documentation/xcode/writing-arm64-code-for-apple-platforms